Dockerfile

```
@@ -26,6 +26,12 @@ RUN \
     yum clean all && \
     rm -rf /var/cache/yum/
 
+# Impatiently ignore the Yum mirrors
+RUN for repo in osg-testing osg-upcoming-testing; do \
+        sed -i 's/\#baseurl/baseurl/; s/mirrorlist/\#mirrorlist/' \
+            /etc/yum.repos.d/$repo.repo; \
+     done
```

Fine as you have it; though of course if you wanted, `sed -i` does accept multiple filenames, too

```suggestion
RUN sed -i 's/\#baseurl/baseurl/; s/mirrorlist/\#mirrorlist/' \
        /etc/yum.repos.d/osg{,-upcoming}-testing.repo
```

---

I like that better, thanks!